html-templates/subtemplates/slate-forms.tpl

```diff
@@ -0,0 +1,97 @@
+{load_templates "subtemplates/forms.tpl"}
+
+{template termField inputName=term label='Term' blankOption='Select' blankValue='' currentOption='Current' currentMasterOption='Current Master' default=null error='' hint='' required=false}
+    <?php
+        $this->scope['options'] = [];
+
+        if ($this->scope['currentOption']) {
+            $this->scope['options']['*current'] = $this->scope['currentOption'];
+        }
+
+        if ($this->scope['currentMasterOption']) {
+            $this->scope['options']['*current-master'] = $this->scope['currentMasterOption'];
```

Comment:
Not sure if these changes should go upstream (adding term options to the term field for `*current` & `*current-master`). 
This could probably be overridden here: https://github.com/SlateFoundation/slate/blob/develop/html-templates/subtemplates/slate-forms.tpl

Reply:
@nbey yeah we should move this upstream